packages/core/test/getNextSnapshot.test.ts

```diff
@@ -18,10 +19,10 @@ describe('getNextSnapshot', () => {
       { count: 0 }
     );
 
-    const init = getInitialSnapshot(logic, undefined);
-    const s1 = getNextSnapshot(logic, init, { type: 'next' });
```

Comment:
this is a test file that's called `getNextSnapshot` but you are swapping *some* `getNexSnapshot` calls to `transition`. You have a different file with `transition` tests so I think this file should be left alone